tests/storage/test_storage.py

```diff
@@ -1226,7 +1226,7 @@ def test_query_dataframe_with_sql(self):
         assert result_df.shape == (1, 2)
 
         assert result_df["string_column"][0] == "A"
-        assert result_df["array_column"][0] == [1, 2, 3]
+        assert tuple(result_df["array_column"][0]) == (1, 2, 3,)
```

Comment:
I don't understand why this test was failing, and this isn't a fix that suggests any understanding, but it does work.